.github/workflows/avm.res.desktop-virtualization.workspace.yml

```diff
@@ -0,0 +1,83 @@
+name: "avm.res.desktop-virtualization.workspace"
+
+on:
+  schedule:
+    - cron: "0 12 1/15 * *" # Bi-Weekly Test (on 1st & 15th of month)
+  workflow_dispatch:
```

Comment:
Would you have the change to attach a pipeline badge as a proof of a successful run?

Reply:
Added pipeline badge to PR.